src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/MessagingMultiNodeTestBase.kt

```diff
@@ -73,6 +73,10 @@ abstract class MessagingMultiNodeTestBase : IntegrationTestBase() {
     hmppsQueueService.findByQueueId("cprdeliusmergeeventsqueue")
   }
 
+  val probationDeleteEventsQueue by lazy {
+    hmppsQueueService.findByQueueId("cprdeliusdeleteeventsqueue")
+  }
```

Comment:
It will probably be worthwhile to purge this queue before running any tests - I think we do that with all the others

Reply:
Will update in further PR's